python/src/robyn/modeling/clustering/cluster_builder.py

```diff
@@ -1,10 +1,38 @@
-from typing import Dict, List
+# pyre-strict
 
+import logging
+from dataclasses import dataclass
+from sys import maxsize
+from typing import Dict, List, Optional
+
+import re
+import numpy as np
 import pandas as pd
-from robyn.modeling.clustering.clustering_config import ClusteringConfig
-from robyn.modeling.entities.clustering_results import ClusteredResult
+from robyn.modeling.clustering.clustering_config import (
+    ClusterBy,
+    ClusteringConfig,
+    DependentVarType,
```

Comment:
shouldn't this be from robyn.data.entities.enums import DependentVarType?

Reply:
good catch, I'll address it. I just took what it auto suggested.